.github/workflows/lint.yml

```diff
@@ -38,6 +38,9 @@ jobs:
           max_attempts: 3
           command: |
             sudo apt-get install -y libclang-dev # required dep for cargo-spellcheck
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: stable
```

Comment:
Why is this needed now?

Reply:
@LesnyRumcajs The latest `cargo-spellcheck` requires rust@1.85.0 to compile, however, this project does not work with 1.85.0

Reply:
I'd say it doesn't make sense to compile `cargo-spellcheck` ourselves at all. Can we use, e.g., [binstall](https://github.com/cargo-bins/cargo-binstall) instead? It should reduce CI time.

Reply:
@LesnyRumcajs CI is using `binstall`, just that `cargo-spellcheck` requires building from source

https://github.com/ChainSafe/forest-explorer/actions/runs/13896707715/job/38878919221?pr=171#step:6:179
> WARN The package cargo-spellcheck v0.15.4 will be installed from source (with cargo)

Reply:
Let's pin the `spellcheck` version, then. `spellcheck` compilation issues shouldn't affect our CI. I can see that the assets for it were not uploaded for our runner architecture - probably some job failed silently. https://github.com/cargo-bins/cargo-quickinstall/releases?q=spellcheck&expanded=true

Reply:
@LesnyRumcajs Fixed. And opened issues in upstream repos

Reply:
Thanks!